src/app/address/controllers/address.js

```diff
@@ -17,7 +17,8 @@ class AddressController extends BaseController {
       } else {
         // edit the chosen address
         address = req.sessionModel.get("chosenAddress");
-        values.addressPostcode = req.sessionModel.get("addressPostcode");
+        values.addressPostcode =
+          address?.postalCode || req.sessionModel.get("addressPostcode");
```

Comment:
This handles the issue where we're displaying the user inputted postcode address rather than the formatted postcode from OS.